src/gui/folderwizard/folderwizardremotepath.cpp

```diff
@@ -361,10 +362,9 @@ void FolderWizardRemotePath::initializePage()
 void FolderWizardRemotePath::showWarn(const QString &msg) const
 {
     if (msg.isEmpty()) {
-        _ui->warnFrame->hide();
-
+        _ui->warningFrame->hide();
     } else {
-        _ui->warnFrame->show();
-        _ui->warnLabel->setText(msg);
+        _ui->warningFrame->show();
+        _ui->warningLabel->setText(QStringLiteral("<html><body><b>%1</b></body></html>").arg(msg));
```

Comment:
Qt rhichtext might be a bit shorter than full html :)

Reply:
done